File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    answer = True
    s = s.lower()
    a,b=0,0
    for i in s:
        if i == 'p':
            a+=1
        elif i=='y':
            b+=1
    if a == b:
        return True
    else:
        return False
    

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;
        s = s.toUpperCase();
        int p = s.length() - s.replace("P","").length();
        int y = s.length() - s.replace("Y","").length();
        
        if(p!=y){
            answer = false;
        }

        return answer;
    }
}